pkg/controller/master/rancher/embedded_rancher.go

```diff
@@ -18,6 +20,10 @@ func (h *Handler) RancherSettingOnChange(key string, setting *rancherv3api.Setti
 		return nil, nil
 	}
 
+	if setting.Name == "internal-cacerts" && setting.Value != "" {
```

Comment:
let's move the `internal-cacerts` name to the constant at `register.go` for better consistency.